crds/roman/schema.py

```diff
@@ -0,0 +1,266 @@
+"""This module defines functions for loading JWST's data model schema files which
+describe reference parameters and their values.   The schema files are used to
+validate reference and rmap parameters to screen out illegal values.
+
+The primary functions provided by this module are:
+
+1. Scraping the cal code data models schema for keyword value enumerations
+2. Defining translations to and from "data model paths" like META.SUBARRAY.NAME and
+"physical keyword values, e.g. FITS" like SUBARRAY.
+
+See the tpn.py and locator.py modules,  as well as crds.certify and crds.rmap,
+and crds.selectors for more information.
+"""
+import re
+
+# ====================================================================================
+
+# from jwst import datamodels   # deferred
+# from . import locate  # deferred
+
+import crds
+from crds.core import log, utils, heavy_client, config
+from crds.core.generic_tpn import TpnInfo
+
+# ====================================================================================
+
+
+# XXXX roman  -- EXP_TYPE value prefixes for Roman
+INSTR_PREFIX = {
+    "wfi" : "WFI_",
+    }
```

Comment:
Does that mean remove line 28?

Reply:
I think removing it is fine.    It's related to a JWST repro scheme I hope will go away so it may go away / not matter.   If it does matter,  I think it will be noticed when other code is ported over from JWST or generalized during the repro effort.